web/wp-content/themes/dctx2/template-parts/content-rapidpass.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-koa5ye.php, but found phpcs-scan-koa5yE.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).